obspy/io/nied/fnetmt/core.py

```diff
@@ -0,0 +1,258 @@
+# -*- coding: utf-8 -*-
+"""
+F-net moment tensor file format support for ObsPy.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org) and Yannik Behr
+:license:
+    GNU Lesser General Public License, Version 3
+    (http://www.gnu.org/copyleft/lesser.html)
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from future.builtins import *  # NOQA @UnusedWildImport
+
+import re
+import uuid
+
+from obspy import UTCDateTime
+from obspy.core.event import (Catalog, Comment, Event,
+                              Origin, Magnitude, FocalMechanism, MomentTensor,
+                              Tensor, NodalPlane, NodalPlanes)
+from . import util
+
+
+class FNETMTException(Exception):
+    pass
+
+
+def _get_resource_id(name, res_type, tag=None):
+    """
+    Helper function to create consistent resource ids.
+    """
+    res_id = "smi:local/fnetmt/%s/%s" % (name, res_type)
+    if tag is not None:
+        res_id += "#" + tag
+    return res_id
+
+
+def _buffer_proxy(filename_or_buf, function, reset_fp=True,
```

Comment:
I agree. I chose to not do it here as we much rather should get this across all modules which is already in some other ticket. Let's hope one of us finds the time to do that soon.